datasets/tutorials/online_plugin.py

```diff
@@ -0,0 +1,45 @@
+# The default online plugin!
+from typing import List, Optional, Union
+
+import pandas as pd
+
+from datasets import DatasetPlugin, Mode
+from datasets.context import Context
+
+
+@DatasetPlugin.register_plugin(constructor_keys={"name"}, context=Context.ONLINE)
```

Comment:
Some duplication here in the naming convention used, thoughts on simplifying? Something like:

```python
from datasets import register_plugin
...
@register_plugin(...)
```

or

```python
from datasets import DatasetPlugin
...
@DatasetPlugin.register(...)
```